docker/Dockerfile.ci_lint

```diff
@@ -29,7 +29,7 @@ RUN bash /install/ubuntu1804_install_python.sh
 
 RUN apt-get update && apt-get install -y doxygen graphviz
 
-RUN pip3 install cpplint pylint==2.4.4 mypy black
+RUN pip3 install cpplint pylint==2.4.4 mypy black==20.8b1
```

Comment:
could you also add this in `python/gen_requirements.py` (the `CONSTRAINTS` global). i've been focused on CI work but will pick that up again soon.

Reply:
Done.